update

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env node
+
+'use strict'
+
+const fs = require('fs')
+const path = require('path')
+
+// This script updates the index files list.js and list.txt in the bin directory,
+// as well as the soljson-latest.js files.
+
+fs.readdir(path.join(__dirname, '/bin'), function (err, files) {
+  if (err) return err
+
+  const parsedList = files
+    .map(function (file) { return file.match(/^soljson-v([0-9.]*)-(.*).js$/) })
```

Comment:
This will be different with future versions, it will actually be a proper semver version string, and the semver comparator can be used as is.